app/models/borrow_policy.rb

```diff
@@ -27,6 +27,14 @@ def complete_name
     "#{name} (#{code})"
   end
 
+  def allow_multiple_holds_per_member?
+    code == "A"
```

Comment:
`uniquely_numbered` should be `false` for A tools, so this could be:

```suggestion
     !uniquely_numbered
```

Reply:
Thanks. I see uniquely_numbered and was wondering if that was the right way to do this.